components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -2616,13 +2636,13 @@ def manage_action_containers(request, action, o_type=None, o_id=None,
 
     form = None
     if action == 'addnewcontainer':
-        # Used within the jsTree to add a new Project, Dataset etc under a
-        # specified parent OR top-level
+        # Used within the jsTree to add a new Project, Dataset, Tag,
+        # Tagset etc under a specified parent OR top-level
         if not request.method == 'POST':
             return HttpResponseRedirect(reverse("manage_action_containers",
                                         args=["edit", o_type, o_id]))
-        if o_type is not None and hasattr(manager, o_type) and o_id > 0:
-            # E.g. Parent o_type is 'project'...
+        if o_type == "project" and hasattr(manager, o_type) and o_id > 0:
```

Comment:
Since the only object we could be creating in the tree at the moment that has the option of also having a parent is a `Dataset` that is the reason for constraining this, correct?